### DISCUSSION TITLE
Beginner in testing. I want example of __init__.py in test package

### DISCUSSION THREAD

**Discussion by AntsForBreakfast** [pytest-dev/pytest#13075] [Q&A]
I heared having __init__.py in test packages is good practice but i never written __init__.py and not sure what supposed to be in it. Maybe a simple code example or explanation would help. Im a bit more of a visual learner, code would be perfect. Thanks

**Comment by webknjaz** [comment]
FWIW, I'm of an opinion that `__init__.py` is actually harmful :man_shrugging:

**Reply by The-Compiler** [reply]
If you disagree with something the [pytest docs recommend](https://docs.pytest.org/en/stable/explanation/goodpractices.html#choosing-an-import-mode), consider opening a new issue with some more background so that the documentation can maybe be amended. But a question by a beginner *already* confused by something really isn't the right place for a "oh what the documentation tells you is maybe bad anyways" without any further explanation.

**Comment by graingert** [comment]
@AntsForBreakfast you actually don't need to add anything into the `__init__.py` file, it's fine to be completely empty

Here's some docs: https://docs.python.org/3/tutorial/modules.html#packages


> The `__init__.py` files are required to make Python treat directories containing the file as packages (unless using a [namespace package](https://docs.python.org/3/glossary.html#term-namespace-package), a relatively advanced feature). This prevents directories with a common name, such as string, from unintentionally hiding valid modules that occur later on the module search path. In the simplest case, `__init__.py` can just be an empty file, but it can also execute initialization code for the package or set the `__all__` variable, described later.
